ecl/hql/hqlexpr.cpp

```diff
@@ -15061,6 +15062,10 @@ extern HQL_API bool hasUnknownTransform(IHqlExpression * expr)
         if (expr->hasAttribute(mergeTransformAtom))
             return true;
         break;
+    case no_call:
```

Comment:
This change is needed for the projection to work. I'm open to other suggestions of where it should be, but some code along these lines is needed, because of:

        case CreateNonEmptyRecordSourceActivity:
            if (hasUnknownTransform(expr))
                complexExtra->preventOptimization();
            break;